server/game/stateWatchers/DamageDealtThisPhaseWatcher.ts

```diff
@@ -0,0 +1,51 @@
+import { StateWatcher } from '../core/stateWatcher/StateWatcher';
+import { StateWatcherName, DamageType } from '../core/Constants';
+import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistrar';
+import { IDamageSource } from '../IDamageOrDefeatSource';
+import Player from '../core/Player';
+import { Card } from '../core/card/Card';
+
+export interface DamageDealtEntry {
+    damageType: DamageType;
+    damageSource: IDamageSource;
+    target: Card;
+}
+
+export type IDamageDealtThisPhase = DamageDealtEntry[];
+
+export class DamageDealtThisPhaseWatcher extends StateWatcher<IDamageDealtThisPhase> {
+    public constructor(
+        registrar: StateWatcherRegistrar,
+        card: Card
+    ) {
+        super(StateWatcherName.DamageDealtThisPhase, registrar, card);
+    }
+
+    public getOpponentsWhoseBasesWereDamagedByPlayer(player: Player) {
```

Comment:
Could we make this a little more generic since there is only one card that needs this specific check? In keeping with the other watchers we could add something like `getDamageDealtByPlayer` which accepts a filter and then provide the additional logic in the call from the Forced Surrender impl.